api/envoy/config/filter/http/common/v1alpha/config.proto

```diff
@@ -0,0 +1,243 @@
+syntax = "proto3";
+
+package envoy.config.filter.http.common.v1alpha;
```

Comment:
super quit drive by nit: Please put this in a more specific namespace/file/directory than "envoy.config.filter.http.common.v1alpha"